### PR TITLE
Move Somgoms cover switch definition

### LINKED ISSUE #4254: Somgoms curtain switch incorrectly identifies as curtain motor
I have a somgoms 3 touch button curtain switch.
https://nl.aliexpress.com/item/4001237395363.html (zigbee curtain option, white)
This is a EU version.

It reports manufacturer code: `_TZE200_sbordckq` which it seems is mapped as a curtain motor?
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/tuya.js

There is however a US version:
https://www.zigbee2mqtt.io/devices/ZSTY-SM-1CTZG-US-W.html

Shouldn't this be mapped to a curtain switch instead?
https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/somgoms.js#L20

(sidenote: pairing is done by pressing and releasing the stop button 5 times and holding it the 6th time)

### PR DESCRIPTION
**Pull request by delneet**
The Somgoms 'sbordckq' incorrectly identified as a cover motor controller.
This moves the definition into the Somgoms vendor file and specifies it as a cover switch.

closes #4254 

- [x] run lint
- [x] run test
- [x] create pr for documentation

### CODE REVIEW

File: devices/somgoms.js [hunk 1]
```diff
@@ -25,6 +25,15 @@ module.exports = [
         toZigbee: [tz.tuya_cover_control, tz.tuya_cover_options],
         exposes: [e.cover_position().setAccess('position', ea.STATE_SET)],
     },
+    {
+        zigbeeModel: ['sbordckq'],
+        model: 'TS0601_switch_cover',
```
Comment (Koenkk) [inline devices/somgoms.js:30]: Is there a better model number written on the device itself or product page? (preferable something that gives results on google)

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/1386

File: devices/somgoms.js (hunk 1 above)
Comment (delneet) [reply devices/somgoms.js:30]: I would have to remove it from the wall to make sure.
But googling revealed it's known by `SM-1CTW-EU`.
So I can name it that.

The similar looking (same line of) devices on the docs page follow the `TS0601_something` format.
[This one](https://www.zigbee2mqtt.io/devices/TS0601_switch_4_gang.html) for example.

What do you prefer?

Comment (Koenkk) [reply devices/somgoms.js:30]: `SM-1CTW-EU` since that is the real model number.

Comment (delneet) [reply devices/somgoms.js]: Done.
I also updated the docs pr.

**Comment by delneet** [comment]
@Koenkk It seems the specific model number is not working after all.
Sorry about this.

```
Info 2022-06-01 22:12:42MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x5c0272fffec200c0","ieee_address":"0x5c0272fffec200c0","status":"started"},"type":"device_interview"}'
Info 2022-06-01 22:12:42MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x5c0272fffec200c0"},"type":"pairing"}'
Debug 2022-06-01 22:12:43Received Zigbee message from '0x5c0272fffec200c0', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"_TZE200_sbordckq","modelId":"TS0601"}' from endpoint 1 with groupID null
Debug 2022-06-01 22:12:43Received Zigbee message from '0x5c0272fffec200c0', type 'readResponse', cluster 'genBasic', data '{"stackVersion":0}' from endpoint 1 with groupID null
Debug 2022-06-01 22:12:43Received Zigbee message from '0x5c0272fffec200c0', type 'readResponse', cluster 'genBasic', data '{"dateCode":""}' from endpoint 1 with groupID null
Debug 2022-06-01 22:12:43Received Zigbee message from '0x5c0272fffec200c0', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID null
Info 2022-06-01 22:12:43Successfully interviewed '0x5c0272fffec200c0', device has successfully been paired
Warning 2022-06-01 22:12:43Device '0x5c0272fffec200c0' with Zigbee model 'TS0601' and manufacturer name '_TZE200_sbordckq' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
Info 2022-06-01 22:12:43MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":null,"friendly_name":"0x5c0272fffec200c0","ieee_address":"0x5c0272fffec200c0","status":"successful","supported":false},"type":"device_interview"}'
Info 2022-06-01 22:12:43MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"0x5c0272fffec200c0","supported":false},"type":"pairing"}'
Debug 2022-06-01 22:12:48Received Zigbee message from '0x5c0272fffec200c0', type 'read', cluster 'genTime', data '["localTime"]' from endpoint 1 with groupID null
Debug 2022-06-01 22:12:52Received Zigbee message from '0x5c0272fffec200c0', type 'commandMcuSyncTime', cluster 'manuSpecificTuya', data '{"payloadSize":10018}' from endpoint 1 with groupID null
```

It seems like it has to be `TS0601` after all.
`data '{"manufacturerName":"_TZE200_sbordckq","modelId":"TS0601"}'`
What is the correct way to get this in the `somgoms.js` file definitions?

What keys do I use? `zigbeeModel`, `model`, `fingerprint` ?

```
{
        zigbeeModel: ['?'],
        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_sbordckq'}],
        model: '?',
        vendor: 'Somgoms'
}
```

**Comment by Koenkk** [comment]
@delneet fixed it: https://github.com/Koenkk/zigbee-herdsman-converters/commit/63731c6637725d30a2f4e60530371af66646831d

**Pull request merged**